docs/DeveloperGuide.adoc

```diff
@@ -800,13 +800,67 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |traveler |add a new person |keep all my important contacts in case of an emergency
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |traveler |edit a person details| so that it is up-to-date
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |traveler |find a person by name | locate details of persons without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |traveler |find a person by tag |  locate details of tags without having to go through the entire list
+
+|`* * *` |businessman |find a person by company |locate details of companies without having to go through the entire list
+
+|`* * *` |traveler |delete a person |remove entries that I no longer need
+
+|`* * *` |traveler |sort persons by name |easily view the details
+
+|`* * *` |traveler |sort persons by tag |easily view the details
```

Comment:
Maybe you might want to specify that this is for a traveler with many contacts, as it would sound more "necessary" with the given context.